packages/manager/modules/pci/src/projects/project/streams/tokens.service.js

```diff
@@ -0,0 +1,96 @@
+import { ACTIONS } from './tokens.constants';
+
+import Token from './token.class';
+
+export default class PciProjectStreamsTokensService {
+  /* @ngInject */
+  constructor($q, OvhApiCloudProjectIo) {
+    this.$q = $q;
+    this.OvhApiCloudProjectIo = OvhApiCloudProjectIo;
+  }
+
+  getAll(projectId, stream) {
+    return this.OvhApiCloudProjectIo
+      .Stream()
+      .Token()
+      .v6()
+      .query({
+        serviceName: projectId,
+        streamId: stream.id,
+      })
+      .$promise
+      .then(tokens => this.$q.all(
```

Comment:
```suggestion
      .then(tokens => Promise.all(
```

Reply:
I would to understand why you recommand that, because:

1. We have a lot of $q in all our codebase, so if we want to switch to Promise, ok, but we need to discuss about that because we can have some effect like 2.

2. In an angularjs context, you can have some effect. Here, unlike $q, Promise is not listening by angularjs. So, if you use Promise, you need to manage digest manually, because angularjs will not be triggered if a value change. It’s why sometimes, in old code, you can see $scope.digest manually for example, that is not a good practice anymore :s

If the main objective is to be agnostic about angularjs, it’s a good idea but not inside a service dedicated to angularjs imo.

Reply:
Related to previous review (https://github.com/ovh-ux/manager/pull/1382#discussion_r329916030).

And also applied to the recent `@ovh-ux/manager-pci` module.

I tends to follow theses newest enhancements.

Should `Promise.all` be avoided for now?

For instance:

```sh
$ git grep 'Promise.all'
packages/manager/modules/pci/src/components/project/images-list/images-list.controller.js:    return Promise.all([
packages/manager/modules/pci/src/components/project/regions-list/regions.service.js:      .then(regions => Promise.all(map(regions, id => this.OvhApiCloudProjectRegion.v6().get({
packages/manager/modules/pci/src/projects/project/edit/remove/remove.controller.js:    return Promise.all(promises)
packages/manager/modules/pci/src/projects/project/failover-ips/onboarding/order/order.routing.js:            .then(regions => Promise.all(map(regions, id => OvhApiCloudProjectRegion.v6().get({
packages/manager/modules/pci/src/projects/project/failover-ips/order/order.routing.js:            .then(regions => Promise.all(map(regions, id => OvhApiCloudProjectRegion.v6().get({
packages/manager/modules/pci/src/projects/project/private-networks/add/add.service.js:    return Promise.all(subnets.map(subnet => this.OvhApiCloudProjectNetworkPrivateSubnet.v6().save({
packages/manager/modules/pci/src/projects/project/quota/quota.service.js:      paymentMethodId => Promise.all(
packages/manager/tools/component-rollup-config/src/plugins/translation-utils.ts:            Promise.all([
```

Reply:
`Promise.all` has some unfortunate consequences as long as we use AngularJS so I think we should avoid them for now :)